tests/Functional/BaseTestCase.php

```diff
@@ -0,0 +1,114 @@
+<?php
+
+namespace Doctrine\Bundle\PHPCRBundle\Tests\Functional;
+
+use Doctrine\Bundle\PHPCRBundle\DataFixtures\PHPCRExecutor;
+use Doctrine\Common\DataFixtures\DependentFixtureInterface;
+use Doctrine\Common\DataFixtures\Loader;
+use Doctrine\Common\DataFixtures\ProxyReferenceRepository;
+use Doctrine\Common\DataFixtures\Purger\PHPCRPurger;
+use Doctrine\ODM\PHPCR\DocumentManager;
+use Symfony\Bridge\Doctrine\DataFixtures\ContainerAwareLoader;
+use Symfony\Bundle\FrameworkBundle\Test\WebTestCase;
+use Symfony\Component\HttpFoundation\Response;
+
+abstract class BaseTestCase extends WebTestCase
+{
+    /**
+     * @var PHPCRExecutor
+     */
+    private $executor;
+
+    protected function getDocumentManager(string $name = null): DocumentManager
+    {
+        if (!self::$kernel) {
+            self::bootKernel();
+        }
+        if (!self::$kernel->getContainer()) {
+            self::$kernel->boot();
+        }
+
+        return self::$kernel->getContainer()->get('doctrine_phpcr')->getManager($name);
+    }
+
+    protected function loadFixtures(array $classNames, $initialize = false)
```

Comment:
todo: instead of doing this in the base case, move https://github.com/symfony-cmf/testing/blob/master/src/Functional/DbManager/PHPCR.php to this repo in src/Test/... (and maybe clean the interface up)